server/src/infra/entities/exif.entity.ts

```diff
@@ -43,6 +54,15 @@ export class ExifEntity {
   @Column({ type: 'float', nullable: true })
   longitude!: number | null;
 
+  @Column({
+    type: 'enum',
+    enum: ProjectionType,
+    enumName: 'projectionType',
+    default: ProjectionType.NONE,
+    nullable: true,
```

Comment:
Do we need `nullable: true` if we specify `default: ProjectionType.NONE`?

Reply:
I am not 100% sure that it is non-nullable in all cases, so I would suggest to keep it nullable.

Reply:
let's set it non-nullable